src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -14,25 +14,25 @@
 
 function Test-AvailableSslOptions
 {
-	$result = Get-AzApplicationGatewayAvailableSslOptions
```

Comment:
@akshaysngupta any reason the prefixes were changed from `Az` back to `AzureRM`?

Reply:
This wasn't intended. I have reverted it back.

Reply:
@akshaysngupta no new commit was pushed to your branch, so this is still an issue